pipelines/nextflow/modules/patch_build/canonical_transcripts.nf

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-process CHECK_PATCH {
+process CANONICAL_TRANSCRIPTS {
```

Comment:
Not too relevant in this case, but processes should start with a verb (not asking to change it in this case).